lib/ReactViews/FeatureInfo/FeatureInfoPanel.jsx

```diff
@@ -200,8 +204,9 @@ function determineCatalogItem(nowViewing, feature) {
         }
 
         for (i = nowViewing.items.length - 1; i >= 0; i--) {
-            if (nowViewing.items[i].dataSource === dataSource) {
-                result = nowViewing.items[i];
+            item = nowViewing.items[i];
+            if (item.dataSource === dataSource || (defined(item._geoJsonItem) && item._geoJsonItem.dataSource === dataSource)) {
```

Comment:
I'm confused about how this works. Isn't item.dataSource the same as item._geoJsonItem.dataSource? It seems like in https://github.com/TerriaJS/terriajs/blob/master/lib/Models/WebProcessingServiceCatalogItem.js#L97 that's what it's returning...